ProteinCartography/fetch_uniprot_metadata.py

```diff
@@ -2,11 +2,14 @@
 import argparse
 import pandas as pd
 import numpy as np
-from bioservices import UniProt
 import os
 import re
-import requests
-from requests.adapters import HTTPAdapter, Retry
+
+from api_utils import (
```

Comment:
Is there a syntactical reason for this to differ from the import call in `fetch_accession.py`?

Reply:
Maybe line length?

Reply:
I initially had more imports but cleaned up the api_utils module more and have far less imports. I'll update it to be on one line.